pytorch_lightning/plugins/environments/cluster_environment.py

```diff
@@ -11,12 +11,42 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+import os
 from abc import ABC, abstractmethod
+from typing import Dict
+
+from pytorch_lightning.utilities import rank_zero_warn
 
 
 class ClusterEnvironment(ABC):
     """ Specification of a cluster environment. """
 
+    DEFAULT_ENVIRON_SETTINGS = {
+        # these are default NCCL settings for communication speedup
+        "NCCL_NSOCKS_PERTHREA": "4",
```

Comment:
is it this one? https://docs.nvidia.com/deeplearning/nccl/user-guide/docs/env.html#nccl-nsocks-perthread
```suggestion
        "NCCL_NSOCKS_PERTHREAD": "4",
```